db/migrate/20220919063831_add_opened_notification_at_to_order_cycle.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class AddOpenedNotificationAtToOrderCycle < ActiveRecord::Migration[6.1]
+  def change
+    add_column :order_cycles, :opened_notification_at, :timestamp
```

Comment:
Is this field on the OC meant to mark that a webhook was called? I'd recommend the name of the column to mention that.
I think this should not be on this OC table but rather on a specific webhook calls table/log. This way we are adding complexity to the critical/core OC table without a real need.

Reply:
Yes, I think I chose this method to keep it simple for the first iteration and get it released.
But when I pick this up next year I'll consider that, it seems like a much better approach.

Reply:
Hmm coming back to this I have a different answer now. We're not intending to log when/if a webhook was called. 
This is a flag to ensure the order cycle "open" state is triggered once, and only once. The application code then schedules any webhooks needing to be delivered. I think  I'll leave this as-is for now.